paddle/framework/operator.cc

```diff
@@ -388,6 +391,20 @@ class RuntimeInferShapeContext : public InferShapeContext {
     }
   }
 
+  VarDesc::VarType GetVarType(const std::string& name) const override {
+    Variable* var = scope_.FindVar(name);
+    if (var->IsType<LoDTensor>()) {
```

Comment:
this logic should be under `Variable`, something like `return var->GetType()`.

Reply:
Done.